services/core-web/src/components/noticeOfWork/applications/NOWSubmissionDocuments.js

```diff
@@ -95,29 +91,6 @@ export const NOWSubmissionDocuments = (props) => {
           }
         />
       </div>
-      <br />
-      <h4>Submission Spatial Files</h4>
```

Comment:
Just curious why this is being removed.

Reply:
It was decided to display all the "application files" in the same table vs separating them. Having spatial files in their own table did not appear to add value. The filter was removed, added a sorter to the table to easily group files by category (there is a picture showing files together in one table if you scroll up)